users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -719,41 +724,354 @@ def run_scf_baseline():
+    ) = get_datasets(pre_process=CodeWrapper("audio_perturb_runner.run"))
+    returnn_args = {
+        "batch_size": 7000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+        "conformer_type": "wei",
+        "specaug_old": {"max_feature": 15},
+        "audio_perturbation": True,
+    }
+    feature_args = {"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2}
+    lr_args = {
+        "peak_lr": 4e-4,
+        "start_lr": 1.325e-05,
+        "end_lr": 1e-5,
+        "increase_epochs": 180,
+        "decrease_epochs": 180,
+        "final_epochs": 0,
+    }
+
+    speeds = [
+        {"prob": 0.6, "minimum": 0.9, "maximum": 1.1},
+        {"prob": 0.6, "minimum": 0.8, "maximum": 1.2},
+        {"prob": 0.6, "minimum": 0.7, "maximum": 1.3},
+        {"prob": 0.5, "minimum": 0.9, "maximum": 1.1},
+        {"prob": 0.5, "minimum": 0.8, "maximum": 1.2},
+        {"prob": 0.5, "minimum": 0.7, "maximum": 1.3},
+        {"prob": 0.4, "minimum": 0.9, "maximum": 1.1},
+        {"prob": 0.4, "minimum": 0.8, "maximum": 1.2},
+        {"prob": 0.4, "minimum": 0.7, "maximum": 1.3},
+    ]
+
+    tempos = [
+        {"prob": 0.4, "minimum": 0.9, "maximum": 1.1},
+        {"prob": 0.4, "minimum": 0.8, "maximum": 1.2},
+        {"prob": 0.4, "minimum": 0.7, "maximum": 1.3},
+        {"prob": 0.5, "minimum": 0.9, "maximum": 1.1},
+        {"prob": 0.5, "minimum": 0.8, "maximum": 1.2},
+        {"prob": 0.5, "minimum": 0.7, "maximum": 1.3},
+        {"prob": 0.6, "minimum": 0.9, "maximum": 1.1},
+        {"prob": 0.6, "minimum": 0.8, "maximum": 1.2},
+        {"prob": 0.6, "minimum": 0.7, "maximum": 1.3},
+    ]
+    preemphases = [
+        {"prob": 0.9, "minimum": 0.9, "maximum": 1.0},
+        {"prob": 0.9, "minimum": 0.8, "maximum": 1.0},
+        {"prob": 0.8, "minimum": 0.9, "maximum": 1.0},
+        {"prob": 0.8, "minimum": 0.8, "maximum": 1.0},
+    ]
+    codecs = [
+        {"encoding": "ULAW", "prob": 0.4},
+        {"encoding": "ULAW", "prob": 0.6},
+    ]
+    non_linearities = [
+        {"prob": 0.4, "alpha": 0.2},
+        {"prob": 0.4, "alpha": 0.4},
+        {"prob": 0.6, "alpha": 0.2},
+        {"prob": 0.6, "alpha": 0.4},
+    ]
+
+    nn_base_args = {}
+
+    for speed in speeds:
+        key_suffix = []
+        # build key suffix
+        key_suffix.append("speed" + f"{speed['prob']}_{speed['minimum']}_{speed['maximum']}")
+        key = f"scf_bs7k_conf-wei-oldspecaug-audio_perturbation_{'_'.join(key_suffix)}"
+        audio_perturb_args = {}
+        audio_perturb_args["speed"] = speed
+        report_args = {
+            "architecture": "conf-wei",
+            "lr": "wei_peak_4e-4_e450_cycle360",
+            "specaug": "wei_adapt_80dim",
+            "wave_norm": "True",
+        }
```

Comment:
I think if this is the baseline, you could also omit them here if you don't want to merge this report with the other reports that did use these args.